### PR TITLE
Connected login page with Redux store

### PR DESCRIPTION
**Pull request by justinjeon**
Files changed: 

- containers/Login/index.tsx
```typescript
 onSuccessHandler = (response: any): void => {
    this.props.onAuthSuccess(
      response.tokenObj.id_token,
      response.tokenObj.access_token
    )
  }
```

```typescript
const mapDispatchToProps = (dispatch: any) => {
  return {
    onAuthSuccess: (id_token: string, auth_code: string) => {
      dispatch(onAuthSuccess(id_token, auth_code))
    },
  }
}

export default connect(
  null,
  mapDispatchToProps
)(Login)
```

 - store/auth/actions.ts (one line at top to disable eslint)

No new components

Expected functionality: Login page now communicates with Redux store

needs to be integrated with Chloe's work

### CODE REVIEW

File: frontend/src/containers/Login/index.tsx [hunk 1]
```diff
@@ -52,8 +54,11 @@ type State = Readonly<typeof initialState>
 class Login extends React.Component<{}, State> {
```
Comment (karenfann) [inline frontend/src/containers/Login/index.tsx]: @chloe-jiang `Props` should be defined (to have `onAuthSuccess` and `onAuthFailure`) and included in `React.Component`

File: frontend/src/containers/Login/index.tsx [hunk 2]
```diff
@@ -100,8 +105,8 @@ class Login extends React.Component<{}, State> {
                 </GoogleSignInButton>
               )}
               clientId={Config.CLIENT_ID}
-              onSuccess={this.responseGoogle}
-              onFailure={this.responseGoogle}
+              onSuccess={this.onSuccessHandler}
+              onFailure={this.onSuccessHandler}
```
Comment (karenfann) [inline frontend/src/containers/Login/index.tsx]: should be changed to appropriate handler `onFailureHandler`

File: frontend/src/containers/Login/index.tsx [hunk 3]
```diff
@@ -131,4 +136,15 @@ class Login extends React.Component<{}, State> {
   }
 }
 
-export default Login
+const mapDispatchToProps = (dispatch: any) => {
+  return {
+    onAuthSuccess: (id_token: string, auth_code: string) => {
+      dispatch(onAuthSuccess(id_token, auth_code))
+    },
```
Comment (karenfann) [inline frontend/src/containers/Login/index.tsx]: @chloe-jiang You imported `onAuthFailure` from `auth/actions`. Now we need to map the function to props within `mapDispatchToProps`. Follow the format for `onAuthSuccess`

**Comment by ramsgoli** [comment]
woohoo!

**Pull request merged**